src/scripts/GameConstants.ts

```diff
@@ -649,6 +649,7 @@ namespace GameConstants {
     ];
 
     export const HoennDungeons = [
+        'Petalburg Woods',
```

Comment:
If you are changing the index for this, all the statistics for dungeons between here and the old index will need to be updated.